yesod-test/Yesod/Test.hs

```diff
@@ -830,6 +831,16 @@ setUrl url' = do
         , rbdGets = rbdGets rbd ++ H.parseQuery (TE.encodeUtf8 urlQuery)
         }
 
+clickOn :: Yesod site => Query -> YesodExample site ()
```

Comment:
Could you add documentation for this, and an `@since` Haddock annotation? Ideally the documentation would include a small example like most of the rest of yesod-test has (You could just copy an example from your tests and that would be fine)